### PR TITLE
fix: upgrade 3rd party lib

### PR DESCRIPTION
**Pull request by ChopperLee2011**
issue:  https://github.com/Wiredcraft/nestjs-bunyan-logger/issues/18

### CODE REVIEW

**Comment by jiangmengyu0409** [comment]
@ChopperLee2011 The CI on node 16 and node 18 generates quite some `EBADENGINE` warning for specific libraries, could you please investigate and see if we need to fix those?

**Comment by ChopperLee2011** [comment]
@jiangmengyu0409 I doubt that we still need to compatible v16 and v18. since this is plan to upgrade 3rd party lib to latest version to fix some vulnerability issues, could we make this a major version upgradation so we can get rid of old nodejs version ?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     timeout-minutes: 25
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [16, 18, 20]
```
Comment (bsdelf) [inline .github/workflows/test.yml:15]: 16 is dead for a while, and 18 will be the next, let's add 22 to the matrix

```
[18, 20, 22]
```

**Comment by jiangmengyu0409** [comment]
> @jiangmengyu0409 I doubt that we still need to compatible v16 and v18. since this is plan to upgrade 3rd party lib to latest version to fix some vulnerability issues, could we make this a major version upgradation so we can get rid of old nodejs version ?

Yes, I think so, and I agree with Frank that we can put 18, 20, 22 instead.

**Review by jiangmengyu0409 — APPROVED** [review]

**Pull request merged**